src/main/java/io/cbitler/stealingartefacts/StealingArtefactsPlugin.java

```diff
@@ -289,7 +301,9 @@ public boolean shouldMarkObject(GameObject object) {
         if (currentState != null && currentState.getDrawerId() != -1) {
             isDrawer = object.getId() == currentState.getDrawerId();
         }
-
+        if (currentState != null && currentState.getLadderId() != -1 && (object.getWorldLocation().distanceTo(currentState.getLadderLocation()) == 0)) {
```

Comment:
Nitpick: we might want to rename this variable to something like 'shouldMark' since it no longer pertains directly to drawers